pkg/workflow/context.go

```diff
@@ -0,0 +1,23 @@
+package workflow
+
+import "context"
+
+// targetCluster is used to store and retrieve a target cluster kubeconfig.
+var targetCluster contextKey = "targetCluster"
+
+// WithTargetCluster returns a context based on ctx containing the target cluster kubeconfig.
```

Comment:
This code illustrates how to populate a context with information as workflows execute.